start-dev.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Keep scripts in a seperate directory, don't pollute the root directory

Reply:
For now it's the only script, when scripts will be added it can be moved along with the new scripts to a separate directory to keep the root dir clean.